core/src/test/java/novoda/lib/sqliteprovider/util/ShadowDatabaseUtils.java

```diff
@@ -1,28 +0,0 @@
-package novoda.lib.sqliteprovider.util;
-
-import android.database.DatabaseUtils;
-
-import com.xtremelabs.robolectric.internal.Implementation;
-import com.xtremelabs.robolectric.internal.Implements;
-
-@Implements(DatabaseUtils.class)
-public class ShadowDatabaseUtils extends com.xtremelabs.robolectric.shadows.ShadowDatabaseUtils {
```

Comment:
Not needed any more, we can use the stock Robolectric `ShadowDatabaseUtils` class